.github/workflows/ci.yml

```diff
@@ -11,9 +11,19 @@ jobs:
     - uses: actions/checkout@v3
     - uses: actions/setup-node@v3
     - name: Install dependencies
-      run: yarn && yarn add -D eslint-config-airbnb-base eslint-plugin-unicorn eslint-plugin-jest@^26.9.0 @typescript-eslint/eslint-plugin @typescript-eslint/parser typescript
+      run: >
+        yarn
+        && yarn add -D eslint-config-airbnb-base eslint-plugin-unicorn eslint-plugin-jest@^26.9.0 @typescript-eslint/eslint-plugin @typescript-eslint/parser typescript github:prettier/prettier#^2.8.1
```

Comment:
This adds prettier@^2.8.1. That means it could also add a later version in the same major branch (like for example 2.12.3). I doubt the fixed git patches will always match and work, so i believe in a future situation the ci will break. Thats's why i suggest using an own fork. I also doubt the issues you opened get fixed soon

Reply:
That is intended, I expect the patches to be integrated and new Prettier released within a month or so, then CI should fail and we should remove the patches :)

Reply:
i actually don't want the ci to fail if it's forseeable... really

Reply:
I have removed the `^` caret symbol, so the Prettier version is fixed